src/types/organisations.d.ts

```diff
@@ -67,6 +67,7 @@ type BaseOrganisationPoll = {
     finished: number
     hasParticipated: boolean
   }
+  funFacts?: FunFacts
```

Comment:
```suggestion
  funFacts?: FunFacts | null
```

If no simulation funFacts will be null